classes/Bed.java

```diff
@@ -0,0 +1,40 @@
+package classes;
+
+public class Bed {
+
+    private double hight;
```

Comment:
```suggestion
    private double height;
```
sry i was wrong last time